functionaltests/src/test/java/specs/scheduler/ScheduledJobFixtureHelper.java

```diff
@@ -86,25 +97,30 @@ public MultiValueResult verifyPaymentIsHandled(String paymentName) throws Except
                 .with("paymentStatus", ofNullable(payment.getPaymentStatus()).map(PaymentStatus::getInterfaceCode).orElse(null))
                 .with("interfaceInteractionsSize", payment.getInterfaceInteractions().size())
                 .with("transactionsSize", payment.getTransactions().size())
+                .with("createdAt", payment.getCreatedAt())
                 .with("lastModifiedAt", payment.getLastModifiedAt())
+                .with("verifiedAt", Instant.now())
                 .with("hasBeenModified", payment.getCreatedAt().compareTo(payment.getLastModifiedAt()) < 0);
     }
 
     /**
      * A maximum duration we wait for a payment is handled by (short) scheduled task.
      * <p>
      * <b>Note:</b><ul>
-     *     <li>this test is depended on actual service's {@link ServiceConfig#getScheduledJobCronForShortTimeFramePoll()}
+     * <li>this test is depended on actual service's {@link ServiceConfig#getScheduledJobCronForShortTimeFramePoll()}
      * value, which is configured by {@code SHORT_TIME_FRAME_SCHEDULED_JOB_CRON} environment variable
      * (or fallback to default one). We expect the short time-frame is "every 30 seconds" (<b>{@code 0/30 * * * * ? *}</b>.
      * Adjust this value if you change short time-frame schedule in the Heroku service.</li>
-     *     <li>We added 10 second gap to let the job complete if any lags occurred.</li>
+     * <li>We almost double this time gap to let the job complete if any lags occurred.
+     * Also, this is important since the scheduled jobs are sequential (e.g. process every tenant queues one-by-one),
+     * hence second tenant transactions processing is blocked in the queue until the first tenant processes all
+     * scheduled transactions.</li>
      * </ul>
      *
-     * @return 40 (seconds)
+     * @return 59 (seconds)
      */
-    public static int waitSecondsTimeout() {
-        return 40;
+    public static long waitSecondsTimeout() {
+        return Math.round(30 * 1.95);
```

Comment:
any explanation for the `"almost double"` mentioned in the Javadoc/`1.95`? and why not just double/`2`?

Reply:
> any explanation for the "almost double"

from the javadoc:

> to let the job complete if any lags occurred. Also, this is important since the scheduled jobs are sequential (e.g. process every tenant queues one-by-one), hence second tenant transactions processing is blocked in the queue until the first tenant processes all transactions.

> why not just double

To avoid the case when the job is executed twice. We kind of want to verify the scheduled job is executed in time, even with lags.

In any case - there are no strict requirements for this test.

Reply:
then please at least add this comment "To avoid the case when the job is executed twice. We kind of want to verify the scheduled job is executed in time, even with lags." in the javadoc.

Reply:
actually, this won't avoid hundred percent. It's just a wish. This test doesn't have such strict requirements, thus i don't want to specify so strict which time frames should be used: i want to leave it flexible for future if we have to adjust the tests again. If i put such comment - in the future it will be a new discussion: is it allowed to break this "avoid executed twice" or not. 
The test is still in "fuzzy" mode and could be changed.

Also, the feature will be removed at all soon - doesn't have sense to spend a lot of time clarifying the tests requirements.

So, i'd better leave it like this.

Reply:
ok, do you think it makes sense to add this information to the Javadoc? that this "won't avoid the problem 100%", and that the values might be needed to adjust again" ?

Reply:
no, i don't think it is necessary in this certain case, especially in view to the fact _the feature will be removed at all soon_